libs/components/forms/src/lib/modules/error/errors.component.ts

```diff
@@ -0,0 +1,56 @@
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+import { CommonModule } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+} from '@angular/core';
+import { ValidationErrors } from '@angular/forms';
+import { SkyIdModule } from '@skyux/core';
+import { SkyI18nModule } from '@skyux/i18n';
+
+import { SkyFormsResourcesModule } from '../shared/sky-forms-resources.module';
+
+import { SkyErrorComponent } from './error.component';
+
+/**
+ * @internal
+ */
+@Component({
+  selector: 'sky-errors',
+  standalone: true,
+  imports: [
+    CommonModule,
+    SkyI18nModule,
+    SkyIdModule,
+    SkyErrorComponent,
+    SkyFormsResourcesModule,
+  ],
+  templateUrl: './errors.component.html',
+  styleUrls: ['./errors.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SkyErrorsComponent {
+  @HostBinding('attr.aria-live')
+  public readonly ariaLive = 'polite';
```

Comment:
I was having difficulty getting the screen reader to read out the errors on the input box code example, here: https://blackbaud.github.io/skyux-pr-preview/1912/code-examples/#/forms/input-box/basic

Reply:
Switched to `role="alert"`, which should make the errors assertive.